src/components/silhouette-pane.scss

```diff
@@ -0,0 +1,51 @@
+@use './vars';
+
+// Make sure to update zoomDuration in silhouette-pane.tsx if you change this.
```

Comment:
FYI: It is possible to import constants from .scss to JavaScript with a bit of configuration, but the comments are fine as well.